exercises/arrays/concat/concat.js

```diff
@@ -21,15 +21,54 @@
  * @param {object[]} rightArray - The right array to concatenate
  * @returns {object[]} The concatenation of the two given arrays
  */
+
+ let dimes = [10,10];
+ let nickels = [5,5];
+ let dimeNickel = ['10','10','5','5'];
+
 function concat(leftArray, rightArray) {
-  // This is your job. :)
+  let left = leftArray.join(",");
```

Comment:
This isn't how you'd concatenate two arrays by hand. Do it like the index card exercises and then transcribe to code.